cognite_toolkit/_cdf_tk/storageio/_base.py

```diff
@@ -136,7 +135,7 @@ def data_to_json_chunk(self, data_chunk: T_WritableCogniteResourceList) -> list[
             A list of dictionaries representing the data in a JSON-compatible format.
 
         """
-        raise NotImplementedError()
+        return data_chunk.dump(camel_case=True)
```

Comment:
This is a good default implementation, it is not needed to force implementing this in each derived class.